src/client/game.cpp

```diff
@@ -3431,12 +3424,12 @@ void Game::handlePointingAtObject(const PointedThing &pointed,
 			runData.object_hit_delay_timer = object_hit_delay;
 		}
 
-		if (input->getLeftClicked())
+		if (isKeyDown(KeyType::DIG))
```

Comment:
`wasKeyDown`?

Reply:
Not sure if that's the way it's supposed to be, but I'll retest after changing this.

Reply:
It is indeed `wasKeyDown`. Fixed.